app/controllers/users_controller.rb

```diff
@@ -1,8 +1,16 @@
 class UsersController < ApplicationController
   # action filter
-  before_action :find_user, only: %i(show)
+  before_action :find_user, except: %i(index create new)
+  before_action :logged_in_user, except: %i(show create new)
```

Comment:
```suggestion
  before_action :logged_in_user, except: %i(show create new)
  before_action :find_user, except: %i(index create new)
```
chưa login thì ko cần load user làm j thanh niên nhá